pkg/api/v1beta1/dynakube/validation/oneagent.go

```diff
@@ -17,14 +18,18 @@ const (
 `
 	errorImageFieldSetWithoutCSIFlag = `The DynaKube's specification tries to enable ApplicationMonitoring mode and get the respective image, but the CSI driver is not enabled.`
 
-	errorNodeSelectorConflict = `The DynaKube's specification tries to specify a nodeSelector conflicts with an another Dynakube's nodeSelector, which is not supported.
-The conflicting Dynakube: %s
-`
+	errorNodeSelectorConflict = `The DynaKube specification attempts to deploy a %s, which conflicts with another DynaKube's %s. Only one Agent per node is supported.
```

Comment:
this does not give the information anymore, that the `nodeSelector` is the problem.

Reply:
That was kind of on purpose, because the `nodeSelector` is not really true. It also pops up if no `nodeSelectors` are specified and 2 dynakubes are colliding. 

So I would rather state that the dynakubes are colliding and which type of agent is the problem. So WDYF, should we still change it back? 🤔

Reply:
only way to fix it is by specifying a node selector, so that should be in the message somewhere

Reply:
Updated